packages/icons/src/autofocus.svg

```diff
@@ -1,9 +1,3 @@
-<!-- 
-  Based on Material design icons by Google - https://github.com/google/material-design-icons
-  which are licensed under the Apache License 2.0. Icon 'crop_free (filled)'.
-
-  File is modified. Modifications are licensed under the MIT license.
--->
```

Comment:
Isn't it based on the Material icon?

Reply:
I missed some parts and just updated the commit.